crates/astria-core/src/protocol/transaction/v1alpha1/mod.rs

```diff
@@ -381,59 +417,69 @@ enum UnsignedTransactionErrorKind {
         raw::UnsignedTransaction::type_url()
     )]
     DecodeAny(#[source] prost::DecodeError),
+    #[error("`actions` field does not make a valid `ActionGroup`")]
```

Comment:
`ActionGroup` is not a concept outside of the crate:
```suggestion
    #[error("`actions` field does not form a valid group of actions")]
```